types/tailwindcss/colors.d.ts

```diff
@@ -0,0 +1,5 @@
+import type { TailwindColorConfig } from './tailwind-config';
+
+declare const colors: TailwindColorConfig;
+
+export default colors;
```

Comment:
```suggestion
export = colors;
```

Reply:
https://github.com/tailwindlabs/tailwindcss/blob/master/colors.js#L1

Reply:
this is for DT definition, not how it will be consumed, you won't expect this to be used as:
```ts
import { default as tailiwngColors } from 'tailwindcss/colors'
```

Reply:
done so thanks